modules/infra/src/main/resources/migrations/V21__new_artifact_fields.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE artifacts
+ADD COLUMN scaladoc_url VARCHAR,
+ADD COLUMN version_scheme VARCHAR,
+ADD COLUMN developers VARCHAR;
```

Comment:
If we want to have a `Seq[Contributor]` instead of a `Option[Seq[Contributor]]` we would need a default value here:

```suggestion
ADD COLUMN developers VARCHAR DEFAULT '[]';
```

Reply:
changed it to `VARCHAR NOT NULL DEFAULT '[]'` 

`ArtifactTableTests` where failing with reason
```bash
[info]     ✕ C17 developers          VARCHAR   (varchar)     NULL      →  String
[info]       Reading a NULL value into String will result in a runtime failure.
[info]       Fix this by making the schema type NOT NULL or by changing the
[info]       Scala type to Option[String] (Checker.scala:56)
```
to fix this added `NOT NULL`.